### PR TITLE
fix(Sheets): prevent content underneath actions

### PR DESCRIPTION
**Pull request by LucidityDesign**

### CODE REVIEW

File: src/components/views/Sheets/Sheets.tsx [hunk 1]
```diff
@@ -169,6 +171,7 @@ const DialogContent = styled(motion(Dialog.Content))<{ size: Size }>`
     inset-inline-start: unset;
     border-end-start-radius: ${({ theme }) => theme.borderRadii.lg};
     border-end-end-radius: ${({ theme }) => theme.borderRadii.lg};
+    display: block;
```
Comment (LucidityDesign) [inline src/components/views/Sheets/Sheets.tsx:174]: This is not needed. However I tried to keep the original display of the component

File: src/components/views/Sheets/Sheets.tsx [hunk 2]
```diff
@@ -230,9 +225,8 @@ const Content = styled.div`
 `
 
 const SmActions = styled.nav`
-  position: fixed;
-  inset-inline: ${({ theme }) => 2 * theme.spacingBase}rem;
-  inset-block-end: ${({ theme }) => 3 * theme.spacingBase}rem;
+  margin-inline: ${({ theme }) => 2 * theme.spacingBase}rem;
+  margin-block: ${({ theme }) => theme.spacingBase}rem ${({ theme }) => 2 * theme.spacingBase}rem;
```
Comment (filiptammergard) [inline src/components/views/Sheets/Sheets.tsx:229]: I would do it like this here which I think is a bit easier to read, but you can choose!

```suggestion
  margin-block-start: ${({ theme }) => theme.spacingBase}rem;
  margin-block-end: ${({ theme }) => 2 * theme.spacingBase}rem;
```

**Review by filiptammergard — CHANGES_REQUESTED** [review]
In this case, the commit message should be a `fix`, to make sure a patch version is released when merging. The other two suggestions are optional. Looks good!

**Review by filiptammergard — APPROVED** [review]
Nice!

**Pull request merged**